controllers/business.js

```diff
@@ -0,0 +1,117 @@
+const { response } = require("express");
+const { Business } = require("../models/business");
+const {
+	addEntity,
+	getEntity,
+	updateEntity,
+	getWithFilter,
+} = require("../database/config");
+
+const healthy = (req, res) => {
+	res.status(200).json({
+		msg: `Healthy`,
+	});
+};
+
+const registerBusiness = async (req, res) => {
```

Comment:
Maybe we could change this to addBusiness to go have the same naming conventions.